### PR TITLE
Add sei to dex.atomic_arbitrages and sandwiches

### PR DESCRIPTION
**Pull request by silointern**
### Description: 
Adding atomic arbitrages from Sei to `dex.atomic_arbitrages`, following the convention of other EVM compatible chains in that directory. Sei dex trades are already included here: https://github.com/duneanalytics/spellbook/blob/main/dbt_subprojects/dex/models/trades/sei/_schema.yml

### CODE REVIEW

**Comment by silointern** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by silointern** [comment]
@hildobby and @jeff-dude - please tag relevant reviewers. Thanks in advance

File: dbt_subprojects/dex/models/sandwiches/sei/dex_sei_sandwiched.sql
Comment (Hosuke) [inline dbt_subprojects/dex/models/sandwiches/sei/dex_sei_sandwiched.sql:1]: We need to define the schema for sandwiches spells as well.

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @silointern 

[Sample dex_sei_atomic_arbitrages]()
[Sample dex_sei_sandwiches](https://dune.com/queries/4325571)
[Sample dex_sei_sandwiched](https://dune.com/queries/4325571)

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**